src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,19 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        String fileText = Files.readString(Paths.get(fileName));
+        StringBuilder sb = new StringBuilder();
```

Comment:
`sb` isn't informative name of the variable

Reply:
I will call